### ISSUE TITLE
Unsupported operation: Infinity or NaN toInt

### ISSUE THREAD

**Issue by Niels-HB** [dnfield/flutter_svg#73]
I got an Unsupported operation exception when using the following SVG.
I commented out the part that I think is causing the issue.
Without it the SVG will display.

SVG:
``` xml
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
   <path d="m18 11.27v-3.77l-4.62-5h-10.38a.5.5 0 0 0 -.5.5v18a.5.5 0 0 0 .5.5h14.5a.5.5 0 0 0 .5-.5v-2.5" fill="none" stroke="#000" />
   <path d="m13.15 3v4.5h4.37" fill="none" stroke="#000" />
   <!-- <path d="m18 11.8a.41.41 0 0 1 .24.08l.59.43h.05.72a.4.4 0 0 1 .39.28l.22.69a.08.08 0 0 0 0 0l.58.43a.41.41 0 0 1 .15.45l-.22.68a.09.09 0 0 0 0 .07l.22.68a.4.4 0 0 1 -.15.46l-.58.42a.1.1 0 0 0 0 0l-.22.68a.41.41 0 0 1 -.38.29h-.79l-.58.43a.41.41 0 0 1 -.24.08.46.46 0 0 1 -.24-.08l-.58-.43h-.06-.72a.41.41 0 0 1 -.39-.28l-.22-.68a.1.1 0 0 0 0 0l-.58-.43a.42.42 0 0 1 -.15-.46l.23-.67v-.02l-.29-.68a.43.43 0 0 1 .15-.46l.58-.42a.1.1 0 0 0 0-.05l.27-.69a.42.42 0 0 1 .39-.28h.78l.58-.43a.43.43 0 0 1 .25-.09m0-1a1.37 1.37 0 0 0 -.83.27l-.34.25h-.43a1.42 1.42 0 0 0 -1.34 1l-.13.4-.35.25a1.42 1.42 0 0 0 -.51 1.58l.13.4-.13.4a1.39 1.39 0 0 0 .52 1.59l.34.25.13.4a1.41 1.41 0 0 0 1.34 1h.43l.34.26a1.44 1.44 0 0 0 .83.27 1.38 1.38 0 0 0 .83-.28l.35-.24h.43a1.4 1.4 0 0 0 1.33-1l.13-.4.35-.26a1.39 1.39 0 0 0 .51-1.57l-.13-.4.13-.41a1.4 1.4 0 0 0 -.51-1.56l-.35-.25-.13-.41a1.4 1.4 0 0 0 -1.34-1h-.42l-.34-.26a1.43 1.43 0 0 0 -.84-.28z" /> -->
   <g stroke="#000">
      <path d="m5.5 10.5h7" fill="none" stroke-linecap="round" />
      <path d="m5.5 13.5h7" fill="none" stroke-linecap="round" />
      <path d="m5.5 16.5h7" fill="none" stroke-linecap="round" />
      <path d="m19.44 13.79a.24.24 0 0 0 -.2.08l-1.64 1.64-.67-.67a.28.28 0 0 0 -.4 0 .28.28 0 0 0 0 .38l.87.87a.28.28 0 0 0 .2.08.27.27 0 0 0 .19-.08l1.84-1.84a.3.3 0 0 0 .07-.3.3.3 0 0 0 -.26-.16z" stroke-width=".25" />
   </g>
</svg>
```

Exception Stacktrace:
```
flutter: ══╡ EXCEPTION CAUGHT BY SVG ╞═══════════════════════════════════════════════════════════════════════
flutter: The following UnsupportedError was thrown resolving a single-frame picture stream:
flutter: Unsupported operation: Infinity or NaN toInt
flutter:
flutter: When the exception was thrown, this was the stack:
flutter: #0      double.toInt (dart:core/runtime/libdouble.dart:178:36)
flutter: #1      double.ceil (dart:core/runtime/libdouble.dart:149:32)
flutter: #2      SvgPathNormalizer._decomposeArcToCubic (package:path_parsing/src/path_parsing.dart:719:71)
flutter: #3      SvgPathNormalizer.emitSegment (package:path_parsing/src/path_parsing.dart:606:14)
flutter: #4      parseSvgPathData (package:path_drawing/src/parse_path.dart:21:16)
flutter: #5      parsePathFromPath (package:flutter_svg/src/svg/xml_parsers.dart:568:10)
flutter: #6      new DrawableSvgShape.parse (package:flutter_svg/src/svg_parser.dart:25:34)
flutter: #7      parseSvgElement (package:flutter_svg/src/svg_parser.dart:64:16)
flutter: #8      Svg.fromSvgString.<anonymous closure> (package:flutter_svg/svg.dart:109:30)
flutter: #9      MappedIterator.moveNext (dart:_internal/iterable.dart:392:18)
flutter: #10     new List.from (dart:core/runtime/libarray_patch.dart:40:17)
flutter: #11     Iterable.toList (dart:core/iterable.dart:398:16)
flutter: #12     Svg.fromSvgString (package:flutter_svg/svg.dart:117:10)
flutter: #13     Svg.fromSvgBytes (package:flutter_svg/svg.dart:83:12)
flutter: <asynchronous suspension>
flutter: #14     Svg.svgPictureDecoder (package:flutter_svg/svg.dart:47:40)
flutter: <asynchronous suspension>
flutter: #15     SvgPicture.svgByteDecoder.<anonymous closure> (package:flutter_svg/svg.dart:446:15)
flutter: #16     AssetBundlePictureProvider._loadAsync (package:flutter_svg/src/picture_provider.dart:424:18)
flutter: <asynchronous suspension>
flutter: #17     AssetBundlePictureProvider.load (package:flutter_svg/src/picture_provider.dart:406:47)
flutter: #18     PictureProvider.resolve.<anonymous closure>.<anonymous closure> (package:flutter_svg/src/picture_provider.dart:306:57)
flutter: #19     PictureCache.putIfAbsent (package:flutter_svg/src/picture_cache.dart:66:22)
flutter: #20     PictureProvider.resolve.<anonymous closure> (package:flutter_svg/src/picture_provider.dart:306:34)
flutter: #21     SynchronousFuture.then (package:flutter/src/foundation/synchronous_future.dart:38:29)
flutter: #22     PictureProvider.resolve (package:flutter_svg/src/picture_provider.dart:304:17)
flutter: #23     _SvgPictureState._resolveImage (package:flutter_svg/svg.dart:549:10)
flutter: #24     _SvgPictureState.didChangeDependencies (package:flutter_svg/svg.dart:523:5)
flutter: #25     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3803:12)
flutter: #26     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #27     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #28     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4901:32)
flutter: #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #31     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #32     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #33     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #34     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #35     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #36     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #37     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #38     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #39     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #40     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #41     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #42     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #43     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #44     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #45     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #46     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #47     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #48     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #49     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #50     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #51     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #52     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #53     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #54     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #55     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #56     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #57     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #58     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #59     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #60     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #61     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #62     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #63     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #64     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #65     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #66     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #67     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #68     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #69     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #70     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #71     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #72     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #73     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #74     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #75     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #76     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #77     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #78     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #79     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #80     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #81     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #82     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #83     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #84     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #85     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #86     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #87     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #88     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #89     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #90     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #91     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #92     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #93     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #94     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #95     Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #96     Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #97     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #98     Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #99     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #100    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #101    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #102    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #103    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #104    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #105    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #106    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #107    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #108    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #109    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #110    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #111    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #112    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #113    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #114    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #115    StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #116    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #117    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #118    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #119    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #120    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #121    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #122    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #123    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #124    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #125    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #126    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #127    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #128    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #129    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #130    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #131    StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #132    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #133    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #134    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #135    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #136    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #137    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #138    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #139    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #140    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #141    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #142    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #143    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #144    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #145    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #146    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #147    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #148    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #149    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #150    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #151    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #152    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #153    SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4796:14)
flutter: #154    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #155    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #156    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #157    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #158    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #159    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #160    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #161    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #162    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #163    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #164    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #165    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #166    ParentDataElement.mount (package:flutter/src/widgets/framework.dart:3993:11)
flutter: #167    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #168    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #169    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3689:16)
flutter: #170    Element.rebuild (package:flutter/src/widgets/framework.dart:3531:5)
flutter: #171    ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3658:5)
flutter: #172    StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3805:11)
flutter: #173    ComponentElement.mount (package:flutter/src/widgets/framework.dart:3653:5)
flutter: #174    Element.inflateWidget (package:flutter/src/widgets/framework.dart:2937:14)
flutter: #175    Element.updateChild (package:flutter/src/widgets/framework.dart:2740:12)
flutter: #176    SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:811:36)
flutter: #177    SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:796:20)
flutter: #178    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2253:19)
flutter: #179    SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:789:11)
flutter: #180    RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:256:23)
flutter: #181    RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:1727:58)
flutter: #182    PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:796:15)
flutter: #183    RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:1727:13)
flutter: #184    RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:245:5)
flutter: #185    RenderSliverMultiBoxAdaptor.insertAndLayoutChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:383:5)
flutter: #186    RenderSliverGrid.performLayout (package:flutter/src/rendering/sliver_grid.dart:583:17)
flutter: #187    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #188    RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:182:11)
flutter: #189    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #190    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:405:13)
flutter: #191    RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1320:12)
flutter: #192    RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:1238:20)
flutter: #193    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #194    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #195    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #196    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #197    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #198    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #199    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #200    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #201    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #202    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #203    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #204    _RenderProxyBox&RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:108:13)
flutter: #205    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #206    RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:798:17)
flutter: #207    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #208    RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:798:17)
flutter: #209    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #210    RenderPadding.performLayout (package:flutter/src/rendering/shifted_box.dart:199:11)
flutter: #211    RenderObject.layout (package:flutter/src/rendering/object.dart:1631:7)
flutter: #212    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:141:11)
flutter: #213    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:339:7)
flutter: #214    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:211:7)
flutter: #215    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:355:14)
flutter: #216    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1506:7)
flutter: #217    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:765:18)
flutter: #218    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:270:19)
flutter: #219    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:674:13)
flutter: #220    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:208:5)
flutter: #221    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
flutter: #222    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
flutter: #223    _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:842:5)
flutter: #224    _invoke (dart:ui/hooks.dart:139:13)
flutter: #225    _drawFrame (dart:ui/hooks.dart:128:3)
flutter:
flutter: Picture provider: ExactAssetPicture(name: "resources/images/icons/icon.svg", bundle: null,
flutter: colorFilter: null)
flutter: Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#553b2(), name:
flutter: "resources/images/icons/icon.svg", colorFilter: null)
```

**Comment by dnfield** [comment]
This is a bug in the path parsing.

I've fixed it there, working on propagating it up to flutter_svg...  Will be published as 0.6.2

**Comment by dnfield** [comment]
Fixed in 1244efd - published to pub as 0.6.2

Thanks for the report!
